src/handler.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+var uniq = require('lodash.uniq');
+
+function Handler(retriever) {
```

Comment:
I think if we address the concern I raised [here](https://github.com/yola/pixabayjs/pull/8#discussion_r33066241) it will be easier to address this issue.